src/test/java/com/github/alexcojocaru/mojo/elasticsearch/LoadElasticsearchDataMojoTest.java

```diff
@@ -41,6 +47,7 @@ protected void setUp() throws Exception
         
         // I cannot find another way of setting the two required propperties at run time.
         mojo.httpPort = esPorts.get(ElasticsearchPort.HTTP);
+
```

Comment:
Addressed in https://github.com/alexcojocaru/elasticsearch-maven-plugin/pull/18/commits/41c9dd39d11de02fdd6aa55a34ac95cc2e79379b